cassandane/Cassandane/Cyrus/Notify.pm

```diff
@@ -142,7 +142,7 @@ sub test_bad
 }
 
 sub test_message
-    :needs_component_idled :min_version_3_9
+    :needs_component_idled :min_version_3_9 :Conversations
```

Comment:
Might also remove the `min_version_3_9`, we decided not to use these anymore AFAIK